packages/sentry-client/src/sentry/constants/status.ts

```diff
@@ -1,6 +1,6 @@
 export enum STATUS {
+	OUTAGE,
```

Comment:
The two basic states for status are `outage` and `available`. I made these states 0 and 1 respectively so that you can check if status is `up` just by evaluating if true or false directly instead of having to know the STATUS enum. (`maintenance` and `issue` are both still technical `up` states which would still resolve true on evaluation.)